nextjs/nextjs-routes.d.ts

```diff
@@ -21,8 +21,8 @@ declare module "nextjs-routes" {
     | StaticRoute<"/api/media-type">
     | StaticRoute<"/api/proxy">
     | StaticRoute<"/api-docs">
-    | DynamicRoute<"/apps/[id]", { "id": string }>
     | StaticRoute<"/apps">
+    | DynamicRoute<"/apps/[id]", { "id": string }>
```

Comment:
Is it an accidental change?

Reply:
This seems to be just a sequence adjustment, automatically completed by the system, and not something I can operate. Do I need to withdraw this modification?

Reply:
I would say yes, since if I run dev build on my machine, I don't see any changes in this file. Weird that it was changed for you.

Reply:
The beginning of the file reads:
// THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
// This file will be automatically regenerated when your Next.js server is running.

Are you sure I need to manually change it back?

Reply:
Well if you don't revert these changes, I guess next time when someone runs dev build it could cause the opposite change in this file.

But ok, let's leave it as it is and see what happens.